api-mssql-go/k8s/authorizationpolicy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: security.istio.io/v1beta1
```

Comment:
is it required for no auth too?
again goats to confirm. If yes, please update the markdown explianing why it is required